drivers/gpio/Kconfig.nct38xx

```diff
@@ -0,0 +1,41 @@
+# NPCX GPIO driver configuration options
+
+# Copyright (c) 2021 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_NCT38XX
+	bool "NCT38XX I2C-based GPIO chip"
+	depends on I2C
+	help
+	  Enable driver for NCT38XX I2C-based GPIO chip.
+
+if GPIO_NCT38XX
+
+config GPIO_NCT38XX_INIT_PRIORITY
+	int "Init priority"
+	default 30
+	help
+	  Device driver initialization priority. The priority should be lower
+	  than I2C device.
```

Comment:
I2C device, alert GPIO device, and gpio_nct38xx may have a different initial level. Directly using BUILD_ASSERT for it may not be a good idea. I only added BUILD_ASSERT between nct38xx devices. Others can be checked at run time level by device_is_ready().